### PR TITLE
Adds glue to the game

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][sprites][feature][add to wiki]

## About the PR 
This PR adds space glue, glue will make you high if you injest it, will apply to tiles to prevent slipping and slow anyone who passes down. Made to be a funny sort of opposite space lube.

## Why's this needed? 
So the assistants can finally snort glue in maint and the clown has yet another weapon to instill horror in the station. :)

## Changelog

```changelog
(u)Hans
(*)Added space glue to the game.
```

### CODE REVIEW

**Comment by colossusqw** [comment]
image
the lube sprite on tiles

File: code/modules/chemistry/Reagents-Misc.dm [hunk 1]
```diff
@@ -3277,16 +3335,16 @@ datum
...
-			name = "glue"
-			id = "glue"
-			description = "Hopefully you weren't the kind of kid to eat this stuff."
-			reagent_state = LIQUID
-			fluid_r = 250
-			fluid_g = 250
-			fluid_b = 250
-			transparency = 255
-			viscosity = 0.7
+		//glue
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-Misc.dm:3338]: Delete

File: code/mob/living/carbon.dm [hunk 2]
```diff
@@ -54,6 +54,13 @@
 						var/atom/target = get_edge_target_turf(src, src.dir)
 						src.throw_at(target, 30, 1, throw_type = THROW_SLIP)
 						random_brute_damage(src, 10)
+		var/turf/T = NewLoc
+		if(T.sticky)
+			if(src.getStatusDuration("slowed")<1)
+				boutput(src, "<span class='notice'>You get slowed down by the sticky floor!</span>")
+			if(src.getStatusDuration("slowed")< 30 SECONDS)
```
Comment (ZeWaka) [inline code/mob/living/carbon.dm:61]: Be consistent with spacing

File: code/modules/chemistry/Reagents-Misc.dm [hunk 3]
```diff
@@ -796,6 +796,64 @@ datum
...
+						else
+							L.changeStatus("slowed", 10 SECONDS, optional = 4)
+				return
+
+			reaction_turf(var/turf/target, var/volume)
+				var/turf/simulated/T = target
+				if(T.sticky == TRUE) return
+				if (istype(T))
+					var/wet = image('icons/effects/water.dmi',"sticky_floor")
+					T.UpdateOverlays(wet, "wet_overlay")
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-Misc.dm:836]: is it really the wet overlay

File: code/modules/chemistry/Reagents-Misc.dm [hunk 4]
```diff
@@ -796,6 +796,64 @@ datum
...
+					T.sticky = TRUE
+					T.wet = 0
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-Misc.dm:838]: why does glue instantly dry water

File: code/modules/chemistry/Reagents-Misc.dm [hunk 5]
```diff
@@ -796,6 +796,64 @@ datum
...
+				. = ..()
+				if(method == TOUCH)
+					var/mob/living/L = M
+					. = 0
+
+					if(L.getStatusDuration("slowed")>100)
+						L.setStatus("staggered", max(M.getStatusDuration("staggered"), 3*volume_passed))
+						if(!ON_COOLDOWN(M, "stuck in glue", 15 SECOND))
+							boutput(M, "<span class='notice'>You get stuck in the glue!</span>")
+					if(L.getStatusDuration("slowed")<100)
```
[the remark this answers is not in the document]
Comment (colossusqw) [reply code/modules/chemistry/Reagents-Misc.dm:824]: 100 is 10 seconds, i should make it 10 * SECONDS

**Comment by TobleroneSwordfish** [comment]
more glue PRs, yes.

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 6]
```diff
@@ -2016,6 +2016,14 @@ datum
 			result_amount = 3
 			mix_phrase = "The substance turns a striking cyan and becomes oily."
 
+		glue
+			name = "Space Glue"
+			id = "glue"
+			result = "glue"
```
Comment (Tarmunora) [inline code/modules/chemistry/Chemistry-Recipes.dm:2022]: does not match reagent

File: code/modules/chemistry/Reagents-Misc.dm [hunk 7]
```diff
@@ -796,6 +796,64 @@ datum
...
+			description = "Industrial superglue that is sure to stick to everything."
+			reagent_state = LIQUID
+			depletion_rate = 0.6
+			fluid_r = 230
+			fluid_b = 60
+			fluid_g = 230
+			transparency = 180
+			viscosity = 0.8
+			block_slippy = 1
+			pierces_outerwear = 1
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-Misc.dm:811]:
```suggestion
```

File: code/modules/chemistry/Reagents-Misc.dm [hunk 8]
```diff
@@ -796,6 +796,64 @@ datum
...
+				. = ..()
+				if(method == TOUCH)
+					var/mob/living/L = M
+					. = 0
+
+					if(L.getStatusDuration("slowed")>10 SECONDS)
+						L.setStatus("staggered", max(M.getStatusDuration("staggered"), 3*volume_passed))
+						if(!ON_COOLDOWN(M, "stuck in glue", 15 SECOND))
+							boutput(M, "<span class='notice'>You get stuck in the glue!</span>")
+					if(L.getStatusDuration("slowed")< 10 SECONDS)
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-Misc.dm:824]:
```suggestion
					else
```

File: code/modules/chemistry/Reagents-Misc.dm [hunk 9]
```diff
@@ -796,6 +796,64 @@ datum
...
+			var/counter
+
+			reaction_mob(var/mob/M, var/method=TOUCH, var/volume_passed)
+				. = ..()
+				if(method == TOUCH)
+					var/mob/living/L = M
+					. = 0
+
+					if(L.getStatusDuration("slowed")>10 SECONDS)
+						L.setStatus("staggered", max(M.getStatusDuration("staggered"), 3*volume_passed))
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-Misc.dm:821]: add time units

File: code/modules/chemistry/Reagents-Misc.dm [hunk 10]
```diff
@@ -796,6 +796,64 @@ datum
...
+						if(!ON_COOLDOWN(M, "stuck in glue", 15 SECOND))
+							boutput(M, "<span class='notice'>You get stuck in the glue!</span>")
+					if(L.getStatusDuration("slowed")< 10 SECONDS)
+						if(volume_passed<25)
+							L.changeStatus("slowed", 4*volume_passed, optional = 4)
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-Misc.dm:826]: add time unit

File: code/modules/chemistry/Reagents-Misc.dm [hunk 11]
```diff
@@ -796,6 +796,64 @@ datum
...
+						else
+							L.changeStatus("slowed", 10 SECONDS, optional = 4)
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-Misc.dm:828]: why not just clamp?

File: code/mob/living/carbon.dm [hunk 12]
```diff
@@ -55,6 +55,13 @@
 						src.throw_at(target, 30, 1, throw_type = THROW_SLIP)
 						random_brute_damage(src, 10)
 
+		var/turf/T = NewLoc
+		if(T.sticky)
+			if(src.getStatusDuration("slowed")<1)
+				boutput(src, "<span class='notice'>You get slowed down by the sticky floor!</span>")
+			if(src.getStatusDuration("slowed")< 30 SECONDS)
+				src.changeStatus("slowed", 2 SECONDS)
```
Comment (Tarmunora) [inline code/mob/living/carbon.dm:63]:
```suggestion
				src.changeStatus("slowed", 2 SECONDS, optional = 2)
```

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/modules/chemistry/Reagents-Misc.dm [hunk 13]
```diff
@@ -796,6 +796,60 @@ datum
...
+
+			reaction_turf(var/turf/target, var/volume)
+				var/turf/simulated/T = target
+				if(T.sticky == TRUE) return
+				if (istype(T))
+					var/wet = image('icons/effects/water.dmi',"sticky_floor")
+					T.UpdateOverlays(wet, "wet_overlay")
+					T.sticky = TRUE
+					T.wet = 0
+					SPAWN(80 SECONDS)
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-Misc.dm:835]: scale with volume

**Review by Tarmunora — CHANGES_REQUESTED** [review]

**Comment by Conchuckter** [comment]
sniffing it should give you brain damage

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**